packages/error-reporting/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "version": "0.1.0",
```

Comment:
If you use the `npm run publish-module` workflow, this should not be set manually.